### ISSUE TITLE
Questions about qustom detection

### ISSUE THREAD

**Issue by w1ngedshadow** [dusty-nv/jetson-inference#1280]
Hello there @dusty-nv , I made my own object detection dataset and custom detection model. Thank you for so informative tutorial. Now I have 2 questions related to it:
1. How I should properly transfer my network to another jetson? Is it enough to transfer the model in .onnx format and labels.txt file to the `ssd/models/transfered_model` on my 2nd  jetson?
2. How can I continue training? I would like to take more photos of objects, add them to the dataset and train model with data that is already exists. Is it possible and how should I do it?
Thank you

**Comment by dusty-nv** [comment]
> 1. Is it enough to transfer the model in .onnx format and labels.txt file to the `ssd/models/transfered_model` on my 2nd  jetson?

Yes that is correct, just copy the onnx file and the labels.txt


> 2\. How can I continue training?

You can use the `--pretrained-ssd` argument and specify the path to the PTH checkpoint that you would like to resume from:
https://github.com/dusty-nv/pytorch-ssd/blob/8ed842a408f8c4a8812f430cf8063e0b93a56803/train_ssd.py#L51
